### PR TITLE
Back button

### PR DESCRIPTION
**Pull request by ronaldsg20**

### CODE REVIEW

File: src/components/create/BtcInputAmount.vue [hunk 1]
```diff
@@ -209,5 +209,12 @@ export default class BtcInputAmount extends Vue {
       this.amountStyle = this.stepState === 'done' ? 'green-box' : 'yellow-box';
     }
   }
+
+  created() {
+    const isInitalValue = this.pegInTxState.amountToTransfer.toBTCString() === '0.00000000';
+    if (!isInitalValue) {
```
Comment (josedahlquist) [inline src/components/create/BtcInputAmount.vue:215]:
```suggestion
    const isInitialValue = this.pegInTxState.amountToTransfer === 0;
    if (!isInitialValue) {
```

File: src/components/create/PegInForm.vue [hunk 2]
```diff
@@ -27,11 +27,17 @@
         </v-col>
       </v-row>
       <v-row class="mx-0">
-        <v-col cols="2" class="d-flex justify-start ma-0 pa-0">
+        <!-- <v-col cols="2" class="d-flex justify-start ma-0 pa-0">
```
Comment (josedahlquist) [inline src/components/create/PegInForm.vue:30]: let's not keep this commented out. Remove it please

unless there is a reason for that, if so, leave the reason commented in the code as well

File: src/components/create/SendBitcoin.vue [hunk 3]
```diff
@@ -206,11 +210,25 @@ export default class SendBitcoin extends Vue {
...
+    }
+    this.clearStore();
+    this.init()
+      .then(() => this.setBtcWallet(wallet));
+  }
+
+  @Emit('back')
+  back() {
+    this.stopAskingForBalance()
+      .then(() => this.$router.push({ name: 'Home' }));
```
Comment (josedahlquist) [inline src/components/create/SendBitcoin.vue:231]: Seems like back event is not required anymore... as this looks like _go home_ event

File: src/components/trezor/SendBitcoinTrezor.vue [hunk 4]
```diff
@@ -0,0 +1,267 @@
+<template>
```
Comment (josedahlquist) [inline src/components/trezor/SendBitcoinTrezor.vue:1]: this seems like a mistake... you are readding the removed sendbitcointrezor component

File: src/services/LedgerService.ts [hunk 5]
```diff
@@ -102,6 +102,9 @@ export default class LedgerService extends WalletService {
             const btc = new AppBtc(transport);
             // eslint-disable-next-line no-restricted-syntax
             for (const item of bundle) {
+              if (this.subscribers.length === 0) {
```
Comment (josedahlquist) [inline src/services/LedgerService.ts:105]: good catch

File: src/services/WalletService.ts [hunk 6]
```diff
@@ -175,6 +176,7 @@ export default abstract class WalletService {
         return;
       }
     }
+    this.loadingBalances = false;
```
Comment (josedahlquist) [inline src/services/WalletService.ts:179]: I recall I suggested wrapping the whole method in a try and setting loadingBalances as false in the finally.

There are a couple other instances above where we should be setting it to false that are not properly covered.

File: src/views/Status.vue [hunk 7]
```diff
@@ -146,14 +146,17 @@
...
+        :txId="txId" :statusData="txData"
+        :showTxId="true" :initialExpand="true"/>
         <v-row justify="center" class="mx-0 mt-5">
           <v-col cols="2" class="d-flex justify-start ma-0 pa-0">
             <v-btn rounded outlined color="#00B520" width="110" @click="back">
               <span>Go home</span>
             </v-btn>
           </v-col>
-          <v-col cols="10" class="d-flex justify-end ma-0 py-0 pl-0">
+          <v-col cols="10" class="d-flex justify-end ma-0 py-0 pl-0 hola hola">
```
Comment (josedahlquist) [inline src/views/Status.vue:159]: hey there... 🖖🏻

File: src/components/create/BtcInputAmount.vue (hunk 1 above)
Comment (hard75) [reply src/components/create/BtcInputAmount.vue]: "this.pegInTxState.amountToTransfer" can't be equal to "0" becacuse it's Satoshi type.

File: src/views/Status.vue (hunk 7 above)
Comment (hard75) [reply src/views/Status.vue:159]: Ready mistake

File: src/components/create/BtcInputAmount.vue (hunk 1 above)
Comment (ronaldsg20) [reply src/components/create/BtcInputAmount.vue]: use `.toNumber()` instead `.toBTCString()`

**Review by josedahlquist — APPROVED** [review]

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Pull request merged**